### PR TITLE
refactor: migrate site to nextjs

### PR DESCRIPTION
**Pull request by chiptus**
This PR handles the migration to nextjs.

## Known bugs and issues:
- [ ] we should use `next/image` for our images instead of `` see https://nextjs.org/docs/messages/no-img-element  
- [ ] currently we export static files, since we plan to deploy to vercel, we can remove that
- [ ] internal docs are not working `./docs/...`
- [x] need to sync with new design
- [x] add site form
- [ ] solve ts issue https://github.com/4tals/LinksForIsrael/pull/269/files#diff-a9fc6dbf9b73b2e4da59f94065c61e0f3d6e58a15a42ccd776d965e0ed07dca6R5
- [x] merge csses https://github.com/4tals/LinksForIsrael/pull/269/files#diff-a9fc6dbf9b73b2e4da59f94065c61e0f3d6e58a15a42ccd776d965e0ed07dca6R5

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (bergden-resonai) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: The changes here seem like some automatic linter, and the output doesn't look correct (same for other .md files)

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      }
+    },
+    {
+      "files": "*.{json,babelrc,eslintrc,remarkrc,prettierrc}",
+      "options": {
+        "useTabs": false
+      }
+    }
+  ]
+}
```
Comment (bergden-resonai) [inline .prettierrc:21]: Missing newline

File: README copy.md
Comment (bergden-resonai) [inline README copy.md:1]: Why is it a `copy`?

Comment (bergden-resonai) [inline README copy.md:1]: Doesn't seem like we need this file (or at least big parts of it)

File: .github/ISSUE_TEMPLATE/feature_request.md
Comment (chiptus) [reply .github/ISSUE_TEMPLATE/feature_request.md:1]: I'll revert it

**Pull request closed without merging**